jekyll/_cci2/runner-installation.adoc

```diff
@@ -0,0 +1,390 @@
+= Installing the CircleCI Runner
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This page describes how to install CircleCI Runner on Linux and macOS platforms.
+
+toc::[]
+
+=== Prerequisites
+
+==== Supported Platforms
+
+Currently, the following platforms are supported with CircleCI Runner:
+
+- Ubuntu 18.04 running on an x86_64 (AMD64) or ARM64 architecture.
+- macOS Catalina (10.15.6+) or above, running on an x86_64 architecture.
+
+CAUTION: Running in a container is not currently supported.
+
+==== Authentication
+
+In order to complete this process you will need to create a namespace, resource-class and authentication token by performing the steps listed below:
+
+. Install the circleci command-line tool.
+
+. Create a namespace for your organization's runner resources under:
+
+NOTE: Each organization can only create a single namespace. If you already use orbs, this namespace will be the same namespace as the orbs use. Use the following command: `circleci namespace create <name> <vcs-type> <org-name>` (e.g. if your GitHub URL is `https://github.com/circleci`, then use: `circleci namespace create my-namespace github circleci`.
+
+. Create a resource class for your runner for your namespace using the following command: `circleci runner resource-class create <resource-class> <description>` (e.g. `circleci runner resource-class create my-namespace/my-resource-class my-description`)
```

Comment:
Worth explaining what a resource class is here? Or linking to a Definitions section?

Reply:
fixed